packages/neuron-ui/src/containers/Main/index.tsx

```diff
@@ -37,6 +37,8 @@ const MainContent = () => {
     [network, networks]
   )
 
+  const isLightClientNetwork = network?.type === 2
```

Comment:
`2` is a magic number, it's better to link to the origin or mention why `2` stands for `light client`

Reply:
The type declared at https://github.com/nervosnetwork/neuron/blob/v0.116.2/packages/neuron-ui/src/types/App/index.d.ts#L190

Reply:
Got it